omnigibson/object_states/robot_related_states.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import omnigibson as og
+from omnigibson.object_states.object_state_base import AbsoluteObjectState, BooleanStateMixin
+
+
+_IN_REACH_DISTANCE_THRESHOLD = 2.0
+
+_IN_FOV_PIXEL_FRACTION_THRESHOLD = 0.05
+
+
+def _get_robot():
```

Comment:
In general, our whole paradigm of robot-related states (e.g.: ToggledOn) is pretty hacky. It'd be nice to think of a cleaner way to pass robot-related info around

Reply:
`obj.states[InHandOfRobot].get_value(robot)`

Reply:
Put these states on robot instead of object

Reply:
Done.